content/source/docs/cloud/api/modules.html.md

```diff
@@ -747,6 +753,12 @@ curl \
 * `DELETE /organizations/:organization_name/registry-modules/:registry_name/:namespace/:name/:provider`
 * `DELETE /organizations/:organization_name/registry-modules/:registry_name/:namespace/:name`
 
+~> **Deprecation warning**: the following endpoints:
+  * `POST /registry-modules/actions/delete/:organization_name/:name/:provider/:version`
+  * `POST /registry-modules/actions/delete/:organization_name/:name/:provider`
+  * `POST /registry-modules/actions/delete/:organization_name/:name`
+are replaced by the above endpoints and will be removed from future versions of the API!.
```

Comment:
There's a funky issue with the list formatting here, and it looks like adding a newline fixes the list but breaks the alert box:

```
~> **Deprecation warning**: the following endpoints:

  * `POST /registry-modules/actions/delete/:organization_name/:name/:provider/:version`
  * `POST /registry-modules/actions/delete/:organization_name/:name/:provider`
  * `POST /registry-modules/actions/delete/:organization_name/:name`
```

I wonder if there's a way we can invoke the inline notification using a block syntax?

Reply:
We could this, which is a bit of a hack but it works:

```
<div class="alert alert-warning" role="alert">
  **Deprecation warning**: the following endpoints:

  * `POST /registry-modules/actions/delete/:organization_name/:name/:provider/:version`
  * `POST /registry-modules/actions/delete/:organization_name/:name/:provider`
  * `POST /registry-modules/actions/delete/:organization_name/:name`

  are replaced by the above endpoints and will be removed from future versions of the API!
</div>
```

<img width="916" alt="Screen Shot 2021-06-29 at 11 52 11 AM" src="https://user-images.githubusercontent.com/6937257/123829457-799c0f80-d8d0-11eb-912d-5616981caac6.png">